examples/Targeting/AddCustomerNegativeCriteria.cs

```diff
@@ -0,0 +1,129 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V2.Common;
+using Google.Ads.GoogleAds.V2.Errors;
+using Google.Ads.GoogleAds.V2.Resources;
+using Google.Ads.GoogleAds.V2.Services;
+using System;
+using static Google.Ads.GoogleAds.V2.Enums.ContentLabelTypeEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V2
+{
+    /// <summary>
+    /// This code example adds various types of negative criteria as exclusions at the customer
+    /// level. These criteria will be applied to all campaigns for the customer.
+    /// </summary>
+    public class AddCustomerNegativeCriteria : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            AddCustomerNegativeCriteria codeExample = new AddCustomerNegativeCriteria();
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            codeExample.Run(new GoogleAdsClient(), customerId);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example adds various types of negative criteria as exclusions " +
+                    "at the customer level. These criteria will be applied to all campaigns " +
+                    "for the customer.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
+        public void Run(GoogleAdsClient client, long customerId)
+        {
+            // Get the CustomerNegativeCriterionServiceClient .
+            CustomerNegativeCriterionServiceClient customerNegativeCriterionService =
+                client.GetService(Services.V2.CustomerNegativeCriterionService);
+
+            // Creates a negative customer criterion excluding the content label type of 'TRAGEDY'.
+            CustomerNegativeCriterion tragedyCriterion = new CustomerNegativeCriterion()
+            {
+                ContentLabel = new ContentLabelInfo()
+                {
+                    Type = ContentLabelType.Tragedy
+                }
+            };
+
+            // Creates a negative customer criterion excluding the placement with url
```

Comment:
Can we change lower case url to URL here?

Reply:
Done